FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN dotnet restore "src/Microsoft.Health.Fhir.Synapse.Tool/Microsoft.Health.Fhir
 
 COPY . .
 WORKDIR "/src/src/Microsoft.Health.Fhir.Synapse.Tool"
-RUN dotnet build "Microsoft.Health.Fhir.Synapse.Tool.csproj" -c Release -o /app/build
+RUN dotnet build "Microsoft.Health.Fhir.Synapse.Tool.csproj" -c Release -o /app/build -p:Version=0.5.0
```

Comment:
Not in this PR, I feel we can use a static place for pipeline version, currently we have version value in 1. ARM Template. 2. Powershell Script. 3. CI Yaml. 4. This Dockerfile

Reply:
+1 let's file an ADO item to track this in github issues